### PR TITLE
fix: default output is to stdout now

### PR DESCRIPTION
**Pull request by mpa-LHutchinson**
### Which problem does the PR fix?

This PR addresses an issue with the project where the user had to specify a file to put the output in. Now, if the user does not provide the --output argument, the output will go to stdout instead.

This is related to #1

### CODE REVIEW

File: yamldifftool.py [hunk 1]
```diff
@@ -35,9 +36,9 @@ def filter_defaults(base, user_provided, root_diff, path=[], strict=False):
     parser.add_argument("default_values", metavar="default_values.yaml", type=str,
                         help="Base values.yaml for the helm chart version you're using")
     parser.add_argument("user_customized_values", metavar="customized.yaml", type=str, help="Customized values.yaml")
-    parser.add_argument("-o", "--output", default="output", type=str,
-                        help="Write to filename. Otherwise default is output")
-    parser.add_argument("-s", "--strict", default=False, action=argparse.BooleanOptionalAction, type=bool,
+    parser.add_argument("-o", "--output", default=None, type=str,
+                        help="Write to filename. If not specified, output is written to stdout.")
+    parser.add_argument("-s", "--strict", default=False, action=argparse.BooleanOptionalAction,
```
Comment (jessesimpson36) [inline yamldifftool.py]: was it intentional to remove `type=bool` from the `--strict` flag on line 41?  Can you provide justification for that change, or change it back?

**Comment by jessesimpson36** [comment]
I confirmed that this change does solve the issue. If you address the above comment, then I will approve.

**Comment by mpa-LHutchinson** [comment]
Hi @jessesimpson36 what needs to be addressed? Sorry if I'm missing something.

**Comment by jessesimpson36** [comment]
On line 41, I see that `type=bool` was removed from the `--strict` flag.  this change seems irrelevant to me. In my review, I wrote asking for justification. ( or to remove the change to that line )

**Comment by mpa-LHutchinson** [comment]
Oh I see. I accidently changed this while figuring out a fix for the issue. I'll correct this mistake and send it in again.

**Review by jessesimpson36 — APPROVED** [review]

**Comment by jessesimpson36** [comment]
Thank you for your help on this @mpa-LHutchinson !

**Pull request merged**